client-sdk/ts-web/core/README.md

```diff
@@ -1,6 +1,6 @@
 # An oasis-core SDK for TypeScript
 
-Developers, see [this getting started guide](docs/getting-started.md).
+Users of this package, see [this getting started guide](docs/getting-started.md).
```

Comment:
we already had a link to a sample envoy configuration. clarifying this link to indicate that users of this package should see the guide there